larray/core/constrained.py

```diff
@@ -327,7 +327,7 @@ class ConstrainedSession(Session, AbstractConstrainedSession, metaclass=ModelMet
         >>> # here the result of the multiplication of the 'population' array by the 'mortality_rate' array
         >>> # is automatically converted to an integer array (printing a warning message)
```

Comment:
is it still the case?

Reply:
About the warning message ? 
No, indeed. Nice catch.